src/Validation/ValueValidator.php

```diff
@@ -0,0 +1,15 @@
+<?php
+
+
+namespace WMDE\Fundraising\Frontend\Validation;
+
+/**
+ * Validate a single, scalar value
+ *
+ * @license GNU GPL v2+
+ * @author Gabriel Birke < gabriel.birke@wikimedia.de >
+ */
+interface ValueValidator {
```

Comment:
Note that we already have an interface with this name at WMDE (in a library used by the Wikibase software). If there is a good alternative for the name, this might be better to avoid potential confusion. No real objection to using `ValueValidator` though.